Makefile

```diff
@@ -39,3 +44,18 @@ format: format_jsonnet
 .PHONY: format_jsonnet
 format_jsonnet: $(JSONNET_FILES)
 	$(JSONNET_DOCKER) $(JSONNETFMT_ARGS) -- $?
+
+compile:
+	jb install
+	$(COMMODORE_CMD)
+
+test: compile test_go test_conftest
+
+test_go:
+	@if [ -f "tests/go/go.mod" ]; then cd tests/go && go test -v ./...; else echo "===> Skipping Go unit tests"; fi
```

Comment:
We either have tests (target present in `Makefile`) or we do not have them. Maintaining those conditionals is complexity we do not need.

We do not have to write a framework here but just an example. Later on, we would use this to update the component template in Commodore. There we could then implement flags to control which type of test will e added.

Reply:
Again, I was thinking already ahead: What if I want to copy-paste this Makefile between components (or having managed by templates/modulesync) and only want to modify as little as possible. It's not needed for this component, yes, but I wanted to showcase how the facility is there to add additional tests relatively easy even after generating the initial scaffolding. But I don't know the bigger plans here so :shrug: